azure-pipelines.yml

```diff
@@ -205,6 +207,8 @@ parameters:
       value: $(KinlySelfTestApiUrl)/testcall
     - name: VhServices:PexipSelfTestNodeUri
       value: $(PexipSelfTestNode)
+    - name: VhServices:EnableMobileSupport
```

Comment:
I know I said this was ok yesterday but could you change this to be the same as the CD pipeline? CI runs will then ensure any issues with the Library set up are caught early.

Reply:
Awesome, thank you!